### ISSUE TITLE
Literate Haskell

### ISSUE THREAD

**Issue by Miguel-Fontes** [JustusAdam/language-haskell#7]
Hi JustusAdam!

Do you have any useful background about what's happening with Literate Haskell syntax?

I've just cloned your repo and looking forward to take a shot at this issue. I'm still getting my head into vscode extension development and it will be my first contact with textmate syntax definitions so, no warrants included.

Thank you!

**Comment by JustusAdam** [comment]
Not really unfortunately.

Feel free to ask me some questions about vscode extensions.
The texmate sytax stuff I only know the basics myself. It was enough to write the definition for the `.cabal` file, but in case of literal Haskell it uses some fancy include syntax that I don't really comprehend myself. I tried copying the definitions from the haskell.texmate bundle but they don't work in vscode for some reason.
Feel free to take a stab at it though.

**Comment by Miguel-Fontes** [comment]
Right.

So, after totally destroying your project 3 times (this is why we are developers! right?) i got to the point below. I must say, kind sir, my writings are somewhat colorful right now.

I've tested with some personal lhs files and everything looks alright until now. Example below.

I've created a PR as i could't find any other problems. Tell me if there's anything else about this issue! Thanks, friend!

![is-it-right-question-mark](https://cloud.githubusercontent.com/assets/15656072/21197084/4453013c-c221-11e6-929d-e4250c61390d.png)
